package.json

```diff
@@ -55,13 +55,13 @@
     "@semantic-release/git": "^9.0.0",
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "^12.1.2",
-    "@types/classnames": "^2.2.11",
+    "@types/classnames": "^2.3.1",
```

Comment:
I do not understand why we have this package as types because https://github.com/JedWatson/classnames#readme
Is written in typescript.